### DISCUSSION TITLE
SelectManyMenu setting ui-state-focus and ui-state-hover

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#735] [PrimeFaces] [answered]
In a SelectOneMenu element, the ui-state-focus class is not set when I select a list entry with the mouse and ui-state-hover is not set when I hover over the list. ui-state-focus is only set when I navigate to the element with the keyboard. In SelectOneMenu elements this works as expected. That's not tragic, but it's not nice either. Is there a way to do this uniformly?

Thank You.

**Comment by melloware** [comment, marked as answer]
Please try 14.0.0-SNAPSHOT I believe this is fixed.

**Reply by hgfrey** [reply]
is version 14.0.0-SNAPSHOT available as maven download? I've got a "not found" error message from our repository.

**Reply by melloware** [reply]
Yep see this for how to use the SNAPSHOT: https://github.com/primefaces/primefaces?tab=readme-ov-file#snapshot

**Reply by hgfrey** [reply]
I'm not able to download the snapshot version. I get the error message:

**Couldn't download artifact: Unable to get dependency information for org.primefaces:primefaces:jar:14.0.0-SNAPSHOT: Failed to retrieve POM for org.primefaces:primefaces:jar:14.0.0-SNAPSHOT: Could not transfer artifact org.primefaces:primefaces:pom:14.0.0-SNAPSHOT from/to temp (https://oss.sonatype.org/content/repositories/snapshots): Transfer failed for https://oss.sonatype.org/content/repositories/snapshots/org/primefaces/primefaces/14.0.0-SNAPSHOT/primefaces-14.0.0-SNAPSHOT.pom**

Using 13.0.4 as version downloads the 13.0.4 jar. what am I doing wrong?

**Reply by melloware** [reply]
Weird can you put this link in your browser?  https://oss.sonatype.org/content/repositories/snapshots/org/primefaces/primefaces/14.0.0-SNAPSHOT/ ?

I see this: 

![image](https://github.com/primefaces/community/assets/4399574/d1da7277-fb6e-4f4b-899a-5047479703ef)

**Reply by hgfrey** [reply]
I can open the page and I see all the artifacts that are also shown above.

**Reply by hgfrey** [reply]
Here is the complete feedback I get from maven.

![image](https://github.com/primefaces/community/assets/33621098/af3c5a97-4957-43d1-8e3c-3dab0e20d9c1)

**Reply by melloware** [reply]
OK you are getting unknown host `oss.sonatype.org` it looks like your DNS Names or hosts file is messed up. That is a perfectly valid host.

See: https://issues.sonatype.org/browse/OSSRH-36365

You have something wrong with your local environment or DNS names.
